domainlab/models/model_dann.py

```diff
@@ -8,21 +8,45 @@
 
 
 def mk_dann(parent_class=AModelClassif):
-    """Instantiate a Deep Adversarial Net (DAN) model
+    """
+    Instantiate a Deep Adversarial Net (DAN) model
+
+    Details:
+        The model is trained to solve two tasks:
+        1. Standard image classification.
+        2. Domain classification.
+        Here for, a feature extractor is adversarially trained to minimize the loss of the image 
+        classifier and maximize the loss of the domain classifier.
+        For more details, see:
+        Ganin, Yaroslav, et al. "Domain-adversarial training of neural networks."
+        The journal of machine learning research 17.1 (2016): 2096-2030.
 
     Args:
         parent_class (AModel, optional): Class object determining the task
         type. Defaults to AModelClassif.
 
     Returns:
         ModelDAN: model inheriting from parent class
-    """
+
+    Input Parameters:
+        list_str_y: list of labels,
+        list_str_d: list of domains,
+        alpha: total_loss = task_loss + $$\\alpha$$ * regularization_loss,
```

Comment:
maybe put a hint here regularization loss here is the domain classification loss?